### ISSUE TITLE
Web UI Broken

### ISSUE THREAD

**Issue by TuRDMaN** [pi-hole/pi-hole#1395]
**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [Y] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [Y] The issue I am reporting can be *replicated*
- [Y] The issue I'm reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**
1

---
**[BUG | ISSUE] Expected Behaviour:**
I expect the Web UI to load up as normal in my browser.

**[BUG | ISSUE] Actual Behaviour:**

My system is a Raspberry Pi 2 running:
Raspbian GNU/Linux 8.0 (jessie)
Linux version 4.4.50+ (dc4@dc4-XPS13-9333) (gcc version 4.9.3 (crosstool-NG crosstool-ng-1.22.0-88-g8460611) ) #970 Mon Feb 20 19:12:50 GMT 2017

I had an issue with my install (a config file got messed up somehow, but I uninstalled and reinstalled before figuring that out), so I uninstalled and started with a clean pihole install. According to the terminal, everything is running as it should be.
But now, when I navigate to the Web UI, a white page with only text shows up. No links, no buttons, just text.
I then tried doing "pihole checkout dev", and instead of the white page with only text, it's just showing the contents of the index.php file

I'm guessing it's an issue with lighttpd, but I have no idea how to go about fixing it. All I know is, a complete uninstall and reinstall of pihole leaves me with the same problem.

Disk space is not an issue. Rebooting does not help.

**[BUG | ISSUE] Steps to reproduce:**

-Uninstall pihole
-Reinstall pihole

**(Optional) Debug token generated by `pihole -d`:**

`5ajsnxy1u5`

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**Comment by Mcat12** [comment]
What is the output of `sudo service lighttpd status`?

**Comment by TuRDMaN** [comment]
D'oh! I thought I had checked that before and it was running, but it actually wasn't now.

`● lighttpd.service - Lighttpd Daemon
   Loaded: loaded (/lib/systemd/system/lighttpd.service; enabled)
   Active: failed (Result: exit-code) since Thu 2017-04-20 18:55:32 UTC; 2min 39s ago
  Process: 673 ExecStart=/usr/sbin/lighttpd -D -f /etc/lighttpd/lighttpd.conf (code=exited, status=255)
  Process: 429 ExecStartPre=/usr/sbin/lighttpd -t -f /etc/lighttpd/lighttpd.conf (code=exited, status=0/SUCCESS)
 Main PID: 673 (code=exited, status=255)

Apr 20 18:55:19 raspberrypi systemd[1]: Starting Lighttpd Daemon...
Apr 20 18:55:26 raspberrypi lighttpd[429]: Syntax OK
Apr 20 18:55:26 raspberrypi systemd[1]: Started Lighttpd Daemon.
Apr 20 18:55:32 raspberrypi lighttpd[673]: 2017-04-20 18:55:26: (network.c.409) can't bind to port:  80 Address already in use
Apr 20 18:55:32 raspberrypi systemd[1]: lighttpd.service: main process exited, code=exited, status=255/n/a
Apr 20 18:55:32 raspberrypi systemd[1]: Unit lighttpd.service entered failed state.`

So I checked what was using Port 80

`sudo netstat -tulpn | grep :80
tcp6       0      0 :::80                   :::*                    LISTEN      737/apache2`

I guess when I did system updates it installed and enabled the apache service. So I've not stopped and disabled it, and everything is working now.

My mistake, thanks for the help!
